arrocke/gloss-translation
# refactor: replace downshift with headless UI

## What has changed



* Replaced `downshift` usage with `@headlessui/react`
* Replaced `AutocompleteInput` with `Autocomplete`

## Connected Issues



Closes #106

## QA Steps


1. Sign in with a translator account
2. Click the down icon for any word
    * A list of options should appear
3. Select an option
    * The new gloss should be saved
4. Type in the autocomplete input
    * The list of options should appear again, filtered by the input value.
5. Type a brand new gloss, then press tab
    * The new gloss should be saved

## Post-Deployment
